datadog_checks_base/datadog_checks/base/utils/db/utils.py

```diff
@@ -471,12 +471,13 @@ class TagManager:
     multiple times.
     """
 
-    def __init__(self) -> None:
+    def __init__(self, tag_normalizer: Optional[Callable[[Union[str, bytes]], str]] = None) -> None:
```

Comment:
Nit, since this is already TagManager, is it worth just shrinking this new parameter name down to `normalizer`? Is the implication that this deals wit tags enough?